switches/http2/http/HttpDispatcher.cpp

```diff
@@ -256,8 +257,23 @@ namespace Aseba
 	void HttpDispatcher::registerHandler(const Handler& handler, const HttpMethod& method, const strings& uriPath)
 	{
 		handlers[method][uriPath] = handler;
+
+		std::stringstream opId;
```

Comment:
It removes the braces from the path element in the URI Template. I admit it is ugly.
For example the default opId for GET /nodes/{node}/variables is `GET-nodes-node-variables`, the same as the one that would be generated by the Stoplight API designer.